README.md

```diff
@@ -96,15 +46,54 @@ Operator               | Description
 
 Function           | Description
 -------------------|------------
-`full`             | Convert an abstract operator to a dense array
 `check_ctranspose` | Cheap check that `A'` is correctly implemented
 `check_hermitian`  | Cheap check that `A = A'`
 `check_positive_definite` | Cheap check that an operator is positive (semi-)definite
+`diag`             | Extract the diagonal of an operator
+`full`             | Convert an abstract operator to a dense array
+`hermitian`        | Determine whether the operator is Hermitian
+`push!`            | For L-BFGS or L-SR1 operators, push a new pair {s,y}
+`reset!`           | For L-BFGS or L-SR1 operators, reset the data
+`shape`            | Return the size of a linear operator
+`show`             | Display basic information about an operator
+`size`             | Return the size of a linear operator
+`symmetric`        | Determine whether the operator is Symmetric
 
 
 ## Other Operations on Operators
 
 Operators can be transposed (`A.'`), conjugated (`conj(A)`) and conjugate-transposed (`A'`).
+Operators can be sliced (`A[:,3], A[2:4,1:5], A[1,1]`), but unlike matrices, they always return
+operators (see differences below).
+
+## Differences
+
+Unlike matrices, an operator never "becomes" a vector or a number. We feel this
+increases compatibility among all operations, but may lead to errors if left
+unchecked.
```

Comment:
"an operator never reduces to a vector or scalar."  I would remove the next sentence.